setup.py

```diff
@@ -100,91 +143,99 @@ def run(self):
+        include_package_data=True,
+        zip_safe=False,
+        scripts=['airflow/bin/airflow'],
+        install_requires=[
+            'alembic>=0.8.3, <0.9',
+            'babel>=1.3, <2.0',
+            'chartkick>=0.4.2, < 0.5',
+            'croniter>=0.3.8, <0.4',
+            'dill>=0.2.2, <0.3',
+            'python-daemon>=2.1.1, <2.2',
+            'flask>=0.10.1, <0.11',
+            'flask-admin>=1.4.0, <2.0.0',
+            'flask-cache>=0.13.1, <0.14',
+            'flask-login==0.2.11',
+            'future>=0.15.0, <0.16',
+            'funcsigs>=0.4, <1',
+            'gitpython>=2.0.2',
+            'gunicorn>=19.3.0, <19.4.0',  # 19.4.? seemed to have issues
+            'jinja2>=2.7.3, <3.0',
+            'markdown>=2.5.2, <3.0',
+            'pandas>=0.15.2, <1.0.0',
+            'pygments>=2.0.1, <3.0',
+            'python-dateutil>=2.3, <3',
+            'requests>=2.5.1, <3',
+            'setproctitle>=1.1.8, <2',
+            'sqlalchemy>=0.9.8',
+            'thrift>=0.9.2, <0.10',
+            'Flask-WTF==0.12'
+        ],
+        extras_require={
+            'all': devel_all,
+            'all_dbs': all_dbs,
+            'async': async,
+            'celery': celery,
+            'crypto': crypto,
+            'devel': devel_minreq,
+            'devel_hadoop': devel_hadoop,
+            'doc': doc,
+            'docker': docker,
+            'druid': druid,
+            'gcp_api': gcp_api,
+            'hdfs': hdfs,
+            'hive': hive,
+            'jdbc': jdbc,
+            'mssql': mssql,
+            'mysql': mysql,
+            'oracle': oracle,
+            'postgres': postgres,
+            'rabbitmq': rabbitmq,
+            's3': s3,
+            'samba': samba,
+            'slack': slack,
+            'statsd': statsd,
+            'vertica': vertica,
+            'ldap': ldap,
+            'webhdfs': webhdfs,
+            'kerberos': kerberos,
+            'password': password,
+            'github_enterprise': github_enterprise,
+            'qds': qds,
+            'cloudant': cloudant
+        },
+        classifiers={
+            'Development Status :: 5 - Production/Stable',
+            'Environment :: Console',
+            'Environment :: Web Environment',
+            'Intended Audience :: Developers',
+            'Intended Audience :: System Administrators',
+            'License :: OSI Approved :: Apache Software License',
+            'Programming Language :: Python :: 2.7',
+            'Programming Language :: Python :: 3.4',
+            'Topic :: System :: Monitoring',
+        },
+        author='Maxime Beauchemin',
+        author_email='maximebeauchemin@gmail.com',
+        url='https://github.com/airbnb/airflow',
+        download_url=(
+            'https://github.com/airbnb/airflow/tarball/' + version),
+        cmdclass={'test': Tox,
+                  'extra_clean': CleanCommand,
+                  },
+    )
+
+
```

Comment:
PEP8 says 2 lines before classes and function definition but not before an if. It's really not shocking though when `landscape.io` starts policing again it will note this as a "code smell". You can get `flake8` to install commit hooks so that you see those on commit. `flake8 --install-hook`